ramls/check-out-by-barcode-request.json

```diff
@@ -24,6 +24,30 @@
       "description": "Service point where the item has been checked out",
       "type": "string",
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "overrideBlocks": {
+      "description": "Blocks that user wants to override",
+      "type": "object",
+      "itemNotLoanableBlock": {
+        "description": "Parameters for overriding of the 'item not loanable' block",
+        "type": "object",
+        "dueDate": {
+          "description": "Due date for a new loan",
+          "type": "string",
+          "format": "date-time"
+        },
+        "required": [
+          "dueDate"
+        ]
+      },
+      "patronBlock": {
+        "description": "Parameters for overriding of the patron block",
+        "type": "object"
```

Comment:
What are the parameters for overriding a patron block?

Reply:
Wrong description, updated, thanks